wpilibcExamples/src/main/cpp/examples/EventLoop/cpp/Robot.cpp

```diff
@@ -54,9 +54,13 @@ class Robot : public frc::TimedRobot {
         // accelerate the shooter wheel
         .IfHigh([&shooter = m_shooter, &controller = m_controller, &ff = m_ff,
                  &encoder = m_shooterEncoder] {
-          shooter.SetVoltage(units::volt_t{controller.Calculate(
-                                 encoder.GetRate(), SHOT_VELOCITY.value())} +
-                             ff.Calculate(SHOT_VELOCITY));
+          shooter.SetVoltage(
+              units::volt_t{controller.Calculate(encoder.GetRate(),
+                                                 SHOT_VELOCITY.value())} +
+              ff.Calculate(
```

Comment:
The feedforward taking a measurement is sus because there's already a P controller for feedback. You can just pass SHOT_VELOCITY for both arguments.

Reply:
Okay.  Hey just curious.  I usually do a local build before posting a change to github.  However I constantly fight the arm simulation tests both C++ and Java.  

Anything I can do other than commenting the failing test out.  Building and then commenting them back in before upload.

Reply:
and I think that does it.  Any others I need to fix.

Reply:
With the CMake C++ build, you can pass `--gtest_filter=TestName.*` to run specific tests. Idk if the Gradle Java build has anything similar.